website/snippets/_sl-deprecation-notice.md

```diff
@@ -0,0 +1,6 @@
+:::info Deprecation of dbt Metrics and the legacy dbt Semantic Layer
+For users on dbt version 1.5 or earlier &mdash; starting on December 15th, 2023, support for dbt Metrics and the old dbt Semantic Layer (v1.5 and below) will end. To benefit from the latest features, upgrade to the most [recent version](/guides/migration/sl-migration) of dbt and embrace the enhanced dbt Semantic Layer, powered by MetricFlow.
```

Comment:
The link text is "upgrade to the most [recent version] of dbt" but the linked doc doesn't appear to contain any instructions for upgrading their dbt version, nor does it link to those instructions at the bottom of the page.

Suggest removing the "Starting" language with "end" in the same sentence so that we're as clear as possible
```suggestion
For users on dbt version 1.5 or earlier &mdash; On December 15th, 2023, support for dbt Metrics and the old dbt Semantic Layer (v1.5 and below) will end. To benefit from the latest features, upgrade to the most [recent version](/guides/migration/sl-migration) of dbt and embrace the enhanced dbt Semantic Layer, powered by MetricFlow.
```

Reply:
i'll clarify this - you need to change you sl configs first before changing your dbt version.